README.md

```diff
@@ -1,3 +1,30 @@
-# fengari-web-cli
+# fengari-web
 
 Provides anything you need to run [Fengari](https://github.com/fengari-lua/fengari) in the browser.
+
+## Building
+
+```bash
+git clone https://github.com/fengari-lua/fengari-web.git
+npm install
+webpack
```

Comment:
Use `npm build` instead of calling webpack directly, as it doesn't rely on people having correct PATH setup.